docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -0,0 +1,210 @@
+[nRF Connect Lock Example Application](../../examples/lock-app/nrfconnect/README.md) onto a CHIP-enabled Thread network. The instructions are also valid for [nRF Connect Lighting Example Application](../../examples/lighting-app/nrfconnect/README.md).
+
+Prerequisites:
+- 2x nRF52840 DK (PCA10056)
+  > :information_source: One nRF52840 DK can be replaced with another [OpenThread Radio Co-Processor](https://openthread.io/platforms/co-processor) compatible device like nRF52840 Dongle
+- 1x smartphone with Android 8+
+- 1x PC with Ubuntu 20.04 and Docker installed, with a spare Wi-Fi card.<br>
+  > :information_source: While the article uses Ubuntu 20.04 as a reference, all the presented steps should be achievable using other popular operating systems.
+
+The essence of the commissioning process is composed of the following pieces:
+1. CHIPTool discovers the accessory device over BLE.
+2. CHIPTool establishes a secure channel to the device over BLE, sends CHIP operational credentials and Thread provisioning data.
+3. The accessory device joins a Thread network.
+
+BLE is only used during the commissioning phase and afterwards IP connectivity between the smartphone and the accessory device is needed to send operational messages. Since a typical smartphone doesn't have Thread radio built-in, extra effort is needed to prepare full-fledged test environment. The article describes how to turn a PC with spare Wi-Fi card and nRF52840 DK into a Thread border router forwarding Wi-Fi packets to the Thread network the accessory device operates in. The below diagram shows connectivity between network components necessary to allow communication between devices running CHIPTool and Lock applications:
+
+```
+               +--------------------+
+               |    Smartphone      |
+     +---------|  Android CHIPTool  |---------+
+     |         +--------------------+         |
+     |                                        | Wi-Fi
+     |                                        |
+     |                                +---------------+  Ethernet  +----------+
+     |                                |      PC       |------------| Internet |
+     |                                +---------------+            +----------+
+     |                                        |
+ BLE |                                        | USB
+     |                                        |
+     |                                +---------------+
+     |                                |  nRF52840 DK  |
+     |                                | OpenThread RCP|
+     |                                +---------------+
+     |                                        |
+     |         +--------------------+         | Thread
+     |         |    nRF52840 DK     |         |
+     +---------|  Lock Application  |---------+
+               +--------------------+
+```
+## Building and flashing OpenThread RCP firmware
+
+<details>
+<summary>Click here to expand</summary>
+<br>
```

Comment:
Not sure I'm a fan of this, as with the current structure it might suggest these steps are optional.

Reply:
I personally think it's much easier to navigate a long document structured like that. Also I was kind of inspired by OpenThread code labs: https://openthread.io/codelabs/openthread-simulation#0. I wonder if we added numbering to the headers, wouldn't it suggest less that these steps are optional?

Reply:
Might be a good idea, let me think about this. What I'm sure of, is that we shouldn't use this "Click here to expand", as it can also hide text, so if you try to search something on the page, you won't find it with the section contracted.

Reply:
Switched to the old-fashioned guide. ;)